### PR TITLE
Dvc pipeline

### PR DESCRIPTION
**Pull request by LuuJustin**
DVC repro/push/pull should work. Added some info in readme. Added remote AWS S3.

### CODE REVIEW

**Comment by dtronmans** [comment]
The pull request looks very good! Only have two comments (which are more questions than comments)

File: src/get_data.py [hunk 1]
```diff
@@ -13,11 +13,23 @@
 load_dotenv()
 
 if REMOTE:
-    s3 = boto3.client('s3', aws_access_key_id=os.getenv("AWS_ACCESS_KEY_ID"),
+    s3 = boto3.client('s3', region_name=os.getenv('AWS_DEFAULT_REGION'), aws_access_key_id=os.getenv("AWS_ACCESS_KEY_ID"),
```
Comment (dtronmans) [inline src/get_data.py:16]: Does the region need to be specified? I thought s3 only needed the access key ID and the secret access key

File: requirements.txt [hunk 2]
```diff
@@ -2,4 +2,6 @@ tensorflow==2.16.1
 boto3==1.34.95
 keras==3.3.3
 numpy==1.26.0
-scikit-learn==1.4.2
\ No newline at end of file
+scikit-learn==1.4.2
+python-dotenv==1.0.1
+dvc-s3==3.2.0
```
Comment (dtronmans) [inline requirements.txt:7]: Shouldn't dvc be there too? Because 'pip install dvc' would be needed to run the pipeline

File: src/get_data.py (hunk 1 above)
Comment (LuuJustin) [reply src/get_data.py:16]: You're right! it is not needed. I have removed it.

File: requirements.txt (hunk 2 above)
Comment (LuuJustin) [reply requirements.txt:7]: It's been added now, good catch.

**Pull request merged**